docs/source/examples/ldaptor_with_upn_bind.py

```diff
@@ -12,7 +12,7 @@
 gn: Bob
 sn: Roberts
 mail: bob@example.org
-homeDirectory: e:\Users\bob
+homeDirectory: e:\\Users\\bob
```

Comment:
They need to be escaped as the  docstring is not marked as row and then python tried o resolve some `\UXXXX` Unicode value :)